### PR TITLE
Webserver PKCS11 Examples

### PR DESCRIPTION
**Pull request by q-nk**
Updating the web server example scripts to the new NetHSM API.

### CODE REVIEW

**Comment by jans23** [comment]
Has [this feedback](https://github.com/Nitrokey/nethsm-pkcs11/pull/50#issuecomment-1831259458) been considered?

File: container/apache/p11nethsm.conf [hunk 1]
```diff
@@ -5,7 +5,7 @@ slots:
     description: Local HSM (docker)
     operator:
       username: "operator"
-      password: "opPassphrase"
+      password: "operatoroperator"
     instances:
       - url: "https://192.168.178.177:8443/api/v1"
```
Comment (mmerklinger) [inline container/apache/p11nethsm.conf:10]: Please update this line to `- url: "https://localhost:8443/api/v1"` to keep it consistent with the value of `HOST` in the generate script.

File: container/nginx/p11nethsm.conf [hunk 2]
```diff
@@ -3,7 +3,7 @@ slots:
     description: Local HSM (docker)
     operator:
       username: "operator"
-      # password: "opPassphrase"
+      password: "operatoroperator"
     instances:
       - url: "https://192.168.3.161:8443/api/v1"
```
Comment (mmerklinger) [inline container/nginx/p11nethsm.conf:8]: Please update this line to `- url: "https://localhost:8443/api/v1"` to keep it consistent with the value of `HOST` in the generate script.

**Review by mmerklinger — CHANGES_REQUESTED** [review]

**Comment by mmerklinger** [comment]
> Has [this feedback](https://github.com/Nitrokey/nethsm-pkcs11/pull/50#issuecomment-1831259458) been considered?

No, the key is still generated locally and upload to the NetHSM. It depends what we want to show with the example.
@jans23 Do you want to have the safer example and the key generated on the NetHSM?

**Comment by mmerklinger** [comment]
@q-nk Please implement the changes to generate the key on the NetHSM.

**Pull request merged**